src/AccessibilityInsights.SharedUx/Controls/InspectTabsControl.xaml

```diff
@@ -55,8 +55,8 @@
                            FontWeight="Bold" 
                            TextWrapping="Wrap"
                            TextTrimming="CharacterEllipsis"
-                           Height="30" 
-                           AutomationProperties.Name="{x:Static Properties:Resources.tbElementAutomationPropertiesName}"/>
```

Comment:
As far as I can tell, setting AutomationProperties.Name here was a bug. All it did was replace the actual text with "Glimpse" for AT users.